### DISCUSSION TITLE
Laptop: Asus Zenbook 14 (UX3405MA-PZ552WS)

### DISCUSSION THREAD

**Discussion by AlanJojo** [linuxmint/discussions#609] [Hardware reviews]
### Releases

Linux Mint 21.x

### Compatibility

Works fine with some minor problems

### Description of the hardware

Note: Using Linux Mint 22 (Option not given in the template. So, I had to select 21.x)

## What works
- Connected two external monitors (2560x1440) and it works well on said resolution along with laptop screen.
The monitor configs are saved once it is set and loaded when monitors are connected again.
- External monitors, keyboard, keyboard backlight, touchpad, printers, external HDDs, Bluetooth, Wi-Fi, camera, internal speakers and microphone all work out of the box. I didn't have to install any external drivers.
- Able to transfer data with my phone using both USB-B and USB-C ports
Tried transferring data with my friend's iPad using USB-C port and it also works
- One complaint that I have heard a lot about Linux is that it drains battery too much. Didn't experience that with Linux Mint on this model. Thank you, to whoever is writing the code for Power management.

## What doesn't work as expected
- Default Display resolution (2880x1800) makes everything on the laptop screen too small.
200% UI Scale (2880x1800) makes everything too large.
Tried fractional scaling (150%). Even though it has a good view, the fractional scaling started causing problems and even caused Cinnamon to crash.
- Asus Numpad doesn't work out of the box.
- The sound from the internal speakers doesn't have the best quality. I think those drivers needs a bit more work.

### Optional - Notes

- For the laptop screen, I am currently using resolution (1680x1050) with 100% scaling. Works well as it is not fractional scaling.
- Asus Numberpad works using this driver - https://github.com/asus-linux-drivers/asus-numberpad-driver.

**Comment by az2oo1** [comment]
mine makes apps unresponsive and other apps takes too long to launch
